### ISSUE TITLE
Accessibility: Everytime a response button is selected, there is a focus loss. Pressing tab forces the user to start from the first actionable link on the top of the Health Bot (in our case call 911).   

### ISSUE THREAD

**Issue by ericmaino** [microsoft/BotFramework-WebChat#3081]
![image](https://user-images.githubusercontent.com/1005609/78627762-25c68600-7847-11ea-9d72-cff4ab3a9e6b.png)

**Actual**
Select “I Accept” or “Yes” or “No” button. There is a focus loss. Actually the focus is on the Health Bot container (as shown in Red rectangle below) but is not visible. Pressing tab brings the focus to call 911 link at the top of the chat window.   

**Expected**
 The focus should be programmatically set to the first line of new text which is displayed as part of selecting “I Accept” button. In this case  “I Accept” text highlighted in “green” below. 
User Impact:  A keyboard and screen reader user is thrown way off context and they have to press “tab” several times to come back to their original place. 

**WCAG Violation**
Focus Order 2.4.3
https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-order.html

Possibly Related #2608

**Comment by corinagum** [comment]
These are accessibility issues of v3 of Web Chat, which has been deprecated and we no longer support unless there are security risks. For accessible Web Chat, please [migrate to v4](https://github.com/Microsoft/BotFramework-WebChat#migrating-from-web-chat-v3-to-v4).

Additionally, these have already been discussed internally. Is there a reason why they are being filed now?

**Comment by compulim** [comment]
The "I Accept text highlighted in green below" you mentioned is not a clickable nor interactive element. Thus, it is not focusable. We do not expect the user to click on the "I Accept text highlighted in green below".

Thus, there are only 3 things left focusable in your screenshot:

- "call 911" link
- "Terms & Conditions" link
- "Privacy Statement" link

And the first one is "call 911", which is on a correct focus order.

**Comment by JorgeCupiV** [comment]
I think the bug referred to the fact that when a user clicked on the "I accept" button at the example. And a new message arrived with or without new 'focusable' items, the focus was lost and set to the start point again instead of following the conversation.
However it looks like this was fixed on v4 as @corinagum mentions.

I'm attaching the next screenshots as examples:
Image 1. The focus is at the text bar for a user to write:
![issue_3081_0](https://user-images.githubusercontent.com/44839253/79280146-4315d880-7e75-11ea-8e85-b34394c0f729.JPG)
Image 2. The user clicks or presses the Tab button on the "Yes" option
![issue_3081_1](https://user-images.githubusercontent.com/44839253/79280148-43ae6f00-7e75-11ea-8986-e9442397f91d.JPG)
Image 3. The focus is now on the "No" option of the first set of buttons and not resetting to the WebChat root. Therefore it is following the correct focus flow.
![issue_3081_2](https://user-images.githubusercontent.com/44839253/79280149-43ae6f00-7e75-11ea-9b22-4fdb9cd92c56.JPG)
